app/ui/window.js

```diff
@@ -25,7 +25,7 @@ module.exports = class Window {
         title: 'Hyper.app',
         // we want to go frameless on Windows and Linux
         frame: process.platform === 'darwin',
-        transparent: process.platform === 'darwin',
+        transparent: false, // revert the transparent bar on mac now with fullscreen fix,
```

Comment:
Unfortunately it appears to break that, setting that to true results in a rather ugly shadow instead...

![screen shot 2018-05-14 at 9 02 21 pm](https://user-images.githubusercontent.com/10497077/39987987-5348b828-57ba-11e8-8fa5-f9d5bf3bc8f4.png)

Not sure what an optimal fix would be, as I assume you wish to retain window transparency.

Reply:
Just thinking, perhaps that top bit can be styled?

Reply:
Yes, I think window transparency is more important than fullscreen looking perfect 👍 Hmm, What do you mean with the top bit can be styled? the traffic light part that has the shadow? I think your attached screenshot looks OK 👍

Reply:
The bar across the top might be styleable, as although that screenshot looks okay, it looks rather ugly during the transition. Not sure if that's handled by electron, or macOS, but if it's election, I should be able to colour it nicely.

Reply:
They seem to show on hover but it's buggy.

I'll take a look later, I need to study now 😓

Reply:
I've taken another look, and that doesn't seem like an ideal way to do this...

I ended up reverting some changes, and discovered that using titlebar hidden instead of hiddeninset fixes these issues, which is odd.

I'm going to play around with Electron, and see if there's something odd going on there...